_data/language-families.yml

```diff
@@ -12,6 +13,7 @@ cpp: Portuguese-based creoles
 crp: Creoles
 cus: Cushitic
 dra: Dravidian
+esx: Eskimo–Aleut
```

Comment:
Language families like this have many different names.

Seems like we'll eventually need to add more info to language families too, just to make them discoverable.